src/components/Facility/FacilityUsers.tsx

```diff
@@ -13,11 +19,14 @@ import useTanStackQueryInstead from "@/Utils/request/useQuery";
 
 export default function FacilityUsers(props: { facilityId: number }) {
   const { t } = useTranslation();
+
+  let usersList: JSX.Element = <></>;
+
   const { qParams, updateQuery, Pagination, resultsPerPage } = useFilters({
     limit: 18,
     cacheBlacklist: ["username"],
   });
-  const [activeTab, setActiveTab] = useState(0);
+  const [activeTab, setActiveTab] = useState("cardView");
```

Comment:
add strong types always

```suggestion
  const [activeTab, setActiveTab] = useState<"card" | "board">("card");
```